openml/tasks/functions.py

```diff
@@ -275,31 +275,43 @@ def __list_tasks(api_call):
     return tasks
 
 
-def get_tasks(task_ids):
-    """Download tasks.
+def get_tasks(task_ids, download_data=True):
+    """Download task representation and data splits optionally.
     This function iterates :meth:`openml.tasks.get_task`.
     Parameters
     ----------
     task_ids : iterable
-        Integers representing task ids.
+        Integers/Strings representing task ids.
+    download_data : bool
+        Option to trigger download of data along with the meta data.
     Returns
     -------
     list
     """
     tasks = []
     for task_id in task_ids:
-        tasks.append(get_task(task_id))
+        tasks.append(get_task(task_id, download_data))
     return tasks
 
 
-def get_task(task_id):
-    """Download the OpenML task for a given task ID.
+def get_task(task_id, download_data=True):
+    """Download the OpenML task representation for a given task ID, optionally
```

Comment:
Could you please try to keep the first line of the docstring to be a single line and add information about optional arguments afterwards?